packages/react-focus/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-focus",
+  "version": "0.0.1",
+  "description": "Focus helpers to be used in React applications.",
+  "private": true,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluent-ui-react"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "start": "just-scripts dev",
+    "start-test": "just-scripts jest-watch",
+    "update-snapshots": "just-scripts jest -u",
+    "just": "just-scripts"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.1.13",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "23.0.0",
+    "@types/react": "16.8.11",
+    "@types/react-dom": "16.8.4",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.13.9",
+    "@uifabric/build": "^7.0.0",
+    "@uifabric/example-app-base": "^7.11.1",
+    "@uifabric/jest-serializer-merge-styles": "^7.0.6",
+    "@uifabric/prettier-rules": "^7.0.4",
+    "@uifabric/tslint-rules": "^7.0.5",
+    "enzyme": "~3.9.0",
+    "enzyme-adapter-react-16": "^1.2.0",
+    "es6-weak-map": "^2.0.2",
+    "office-ui-fabric-react": "^7.81.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.2",
+    "@uifabric/utilities": "^7.11.0",
+    "@uifabric/merge-styles": "^7.8.1",
+    "tslib": "^1.7.1"
```

Comment:
This has to be `^1.10.0` to prevent `__spreadArray` exceptions per #11739. Did this come from `create-package` or a template? If so please let me know, it has to be fixed.

Reply:
Yes I believe this was from the template.

Reply:
I ran `yarn create-package` and it did use tslib `^1.10.0` correctly, maybe this was created before my PR was merged. If you used `create-package` and ran into any issues, let me know and I'll queue them up. I'll have to update it anyways per upcoming CI changes (project references.)